### PR TITLE
Fix & Test DNS Resource

### PR DESCRIPTION
**Pull request by chriskuchin**

### CODE REVIEW

File: compose.yaml [hunk 1]
```diff
@@ -5,9 +5,9 @@ services:
       - "3000:3000"
     volumes:
       - ./container/:/var/lib/bowtie
+    env_file: config.env
```
Comment (tylerjl) [inline compose.yaml:8]: Didn't know it could do this; nice

File: internal/bowtie/resources/dns.go [hunk 2]
```diff
@@ -318,6 +347,8 @@ func (d *dnsResource) Update(ctx context.Context, req resource.UpdateRequest, re
 		return
 	}
 
+	// servers := mergeServerDetails(plan.Servers, )
```
Comment (tylerjl) [inline internal/bowtie/resources/dns.go:350]: Just checking, you want to keep this around for later?

File: internal/bowtie/resources/policy.go [hunk 3]
```diff
@@ -1,168 +1,168 @@
...
-}
-
-func (p *policyResource) Delete(ctx context.Context, req resource.DeleteRequest, resp *resource.DeleteResponse) {
-
-}
-
-func (p *policyResource) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
-
-}
+// import (
```
Comment (tylerjl) [inline internal/bowtie/resources/policy.go:3]: I think this is the right call, I like having the linter step around

File: internal/bowtie/test/dns_test.go [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+	"strings"
+	"testing"
+	"text/template"
+
+	"github.com/bowtieworks/terraform-provider-bowtie/internal/bowtie/provider"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/plancheck"
+)
+
+func TestAccDNSResource(t *testing.T) {
```
Comment (tylerjl) [inline internal/bowtie/test/dns_test.go:13]: Sweet, I'm glad the acceptance tests are working

**Review by tylerjl — APPROVED** [review]
LGTM; I left a few scattered comments and one question about whether one commented line should be there or not but otherwise looks good.

File: compose.yaml (hunk 1 above)
Comment (chriskuchin) [reply compose.yaml:8]: yeah finch won't work without this since it can't read the env

File: internal/bowtie/resources/dns.go (hunk 2 above)
Comment (chriskuchin) [reply internal/bowtie/resources/dns.go:350]: Nope I will remove it

**Comment by chriskuchin** [comment]
@tylerjl pushed a removal of the comments

**Pull request merged**